gke-deploy/gke-deploy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: gke-deploy
+spec:
+  inputs:
+    # resources:
+    # - name: source-repo
```

Comment:
is this right? the docs for the task mention the git resource but it's commented out here.

Reply:
Thanks for catching that. That was leftover from some experimenting. Fixing now.